raiden/ui/cli.py

```diff
@@ -325,6 +325,16 @@ def options(func):
             default='udp',
             show_default=True
         ),
+        option(
```

Comment:
It would be nice if the user could also use the network name (and we already have that mapping in the code) instead of only the id.

The implementation of a `click` type for this would be very similar to the [`NATChoiceType`](https://github.com/raiden-network/raiden/blob/5790a19ddedd03eec1298388a7c52241b2e2b78e/raiden/utils/cli.py#L229) we already have.

Reply:
I updated to your suggestion. It's way user-friendly now. Please have a look at the implementation though.

Reply:
Nice!

I added one more small suggestion below, other than that LGTM.

Reply:
Fixed as well.